configure.ac

```diff
@@ -730,6 +730,44 @@
         AC_MSG_RESULT(no)
     fi
 
+    if test "x$pcre_jit_works" = "xyes"; then
+
+       AC_MSG_CHECKING(for PCRE JIT EXEC support usability)
+       AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
+                          #include <pcre.h>
+                          #include <string.h>
+                          ]],
+           [[
+           const char* regexstr = "(a|b|c|d)";
+           const char* teststr = "apple";
+           pcre *re;
```

Comment:
can we compress this by declaring the vars on first use. Also maybe directly pass the regex and test data into the funcs? Could safe us ~6 lines?

Reply:
Will do.